pubspec.yaml

```diff
@@ -1,15 +1,16 @@
 name: path
 version: 1.6.3
```

Comment:
You'll have to bump this to 1.6.4 since 1.6.3 got out the door before I saw this. :(

Reply:
Will do. Going to add an example so https://pub.dev/packages/path#-analysis-tab- is happy